provision/provision-site.sh

```diff
@@ -102,6 +133,10 @@ function vvv_provision_site_nginx_config() {
   if [ 'php' != "${NGINX_UPSTREAM}" ] && [ ! -f "/etc/nginx/upstreams/${NGINX_UPSTREAM}.conf" ]; then
     vvv_error " * Upstream value '${NGINX_UPSTREAM}' doesn't match a valid upstream. Defaulting to 'php'.${CRESET}"
     NGINX_UPSTREAM='php'
+    DEFAULTPHP=$(vvv_get_site_config_value 'php' "")
+    if [ '' != "${DEFAULTPHP}" ]; then
+      NGINX_UPSTREAM=$DEFAULTPHP
+    fi
   fi
```

Comment:
```suggestion
  else
    # if php: is configured, set the upstream to match
    if [ '' != "${DEFAULTPHP}" ]; then
      # TODO: Check this version of PHP is actually installed
      NGINX_UPSTREAM="${DEFAULTPHP}"
    fi
  fi
```

Reply:
also, aren't PHP versions of the form `8.0` but nginx upstreams of the form `php80`? We have to do some conversion

Reply:
you are right, fixed in a commit